gfootball/env/observation_processor.py

```diff
@@ -111,11 +158,14 @@ def get_frame(trace):
         field_coords=True,
         color=(0, 255, 0))
     letter = 'H'
+    # Goalkeeper
+    if player_idx == 0:
+      letter = 'G'
```

Comment:
I think we should use team_roles to check if a player is Goal Keeper or not. See https://github.com/google-research/football/blob/master/gfootball/doc/observation.md